nautobot/ipam/migrations/0031_ipam__prefix__add_parent.py

```diff
@@ -43,6 +45,64 @@ def reverse_it(apps, schema_editor):
         vrf.delete()
 
 
+def migrate_ipaddress_status_to_type(apps, schema_editor):
+    """Migrate IPAddress.status DHCP/SLAAC to types."""
+    Status = apps.get_model("extras", "Status")
+    IPAddress = apps.get_model("ipam", "IPAddress")
+    ContentType = apps.get_model("contenttypes", "ContentType")
+
+    ipaddress_ct = ContentType.objects.get_for_model(IPAddress)
+    statuses = Status.objects.filter(content_types=ipaddress_ct)
+    status_active = statuses.get(name="Active")
```

Comment:
We can't assume "Active" is a valid status since they can be changed. If you can't find one, either recreate it, create one called something like "unknown", or pick a different status.

Reply:
I replaced it with a "Migrated" status.